### ISSUE TITLE
Get Error "Your program stop with error code: -2147483645"

### ISSUE THREAD

**Issue by ShaliniNeo** [OpenCppCoverage/OpenCppCoverage#40]
**Before creating an issue, please make sure the answer is not in the [FAQ](https://github.com/OpenCppCoverage/OpenCppCoverage/wiki/FAQ).**

**This template is applied for all bug requests. Remove it for other general question or feature request.**

## Describe your environment
- Version of OpenCppCoverage: Latest
- Architecture (x86/ x64): x64
- Windows version: Windows 7 with SP1

## Describe your problem
Steps to reproduce:
1. Execute OpenCPPCoverage with options for [modules.]
[LastCoverageResults.log](https://github.com/OpenCppCoverage/OpenCppCoverage/files/1856203/LastCoverageResults.log)

Could you reproduce the issue on a toy example?

## Attached file
Attach the log file *LastCoverageResults.log*, located in your current folder.

If the report does not contain the expected files, please run OpenCppCoverage with ```--verbose``` flag. This will add more information into *LastCoverageResults.log*.

**Comment by amai2012** [comment]
https://github.com/OpenCppCoverage/OpenCppCoverage/releases/tag/release-0.9.7.0 says:

> On failure, OpenCppCoverage returns exit code 0x9F8C8E5C.

See #8

**Comment by OpenCppCoverage** [comment]
Hello,

Thanks amai2012 for the answer!

From LastCoverageResults.log: ``Your program stop with error code: -2147483645`` means ``DMDBTestUD.exe`` returned the exit code -2147483645 (0x80000003). OpenCppCoverage emphasized that your program returned an exit code not equal to 0.

Note: In case of internal failure, OpenCppCoverage returns ``0x9F8C8E5C`` (-1618178468).

I hope it will help,

OpenCppCoverage

**Comment by ShaliniNeo** [comment]
How can I troubleshoot this error? From the logs that Open CPP Coverage tool generated, there is not much to infer.

Regards,

**Comment by ShaliniNeo** [comment]
What is the correct way of using options? The help given does not give much information.
In my solution I have multiple projects. These projects have different folders containing .cpp and header files. One of the project output is the Unit test executable which I am running with OpenCPPCoverage to check on the code coverage. How can I give multiple modules to have coverage on. What is "modules" option connection with "sources" option.  

I have tried with various permutations of these options. Only the below option does not throw error that I referred in my previous mail -

OpenCPPCoverage --modules DMBusinessModelU.dll --source
s C:\HemUA\SourceCode\dxhvic_i_main\DM\Barracuda\Console\DMBusinessModel\*.cpp -
-excluded_sources * -- DMDBTestU.exe /a

But the above command does not give me any coverage.

Also which configuration of my executable should run with OpenCPPCoverage? - Debug or Release.

Looking forward to your help.

Regards,
Shalini
